web/controllers/rules/helpers.ex

```diff
@@ -18,4 +18,16 @@ defmodule Dwylbot.Rules.Helpers do
       :test -> test
     end
   end
+
+  @doc """
+  iex> Dwylbot.Rules.Helpers.label_member?([%{"id" => 1, "name"=> "lb"}], "lb")
+  true
+  iex> Dwylbot.Rules.Helpers.label_member?([%{"id" => 1, "name" => "lb"}], "er")
+  false
+  """
```

Comment:
Beautiful, if you change `doctest Dwylbot.Rules.Helpers` to `doctest Dwylbot.Rules.Helpers, import: true`. Then it will import `label_member?` and the tests can just be:

```elixir
iex> label_member?([%{"id" => 1, "name"=> "lb"}], "lb")
true
```